yt/geometry/coordinates/geographic_coordinates.py

```diff
@@ -158,7 +159,11 @@ def _dlatitude_to_dtheta(field, data):
 
         def _longitude_to_phi(field, data):
             # longitude runs from -180 to 180
-            return (data[("index", "longitude")] + 180) * np.pi / 180.0
+            lonvals = data[("index", "longitude")]
+            neglons = lonvals < 0.0
+            if np.any(neglons):
+                lonvals[neglons] = lonvals[neglons] + 360.0
```

Comment:
Oh, just realized that this could all be replaced with 

```suggestion
            lonvals = np.mod(lonvals + 360., 360.)
```

I don't have much opinion on whether it's better -- it's nice to not have to create the mask, nice to have one line of code instead of 3, but it's annoying to have the unnecessary operations on data in the range 0, 180. Anyone have a vote one way or the other? I'd probably just leave it as-is, but mainly cause the jenkins suite finally ran....